modevo-consistency/src/main/java/giis/modevo/consistency/OracleCsv.java

```diff
@@ -0,0 +1,345 @@
+			for (int j = i + 1; j < matrixResultSet.length && j <= end; j++) {
+				sortIteration(numeric, matrixResultSet, rowsCassandra, i, j, numberColumn); //Sorts two rows
+			}
+		}
+		return matrixResultSet;
+	}
+
+	/**
+	 * Auxilary method of orderAlgorithm that sorts two rows in descending order
+	 */
+	private void sortIteration (boolean numeric, String[][] matrixResultSet, List<String> rowsCassandra, int row1, int row2,
+			int numberColumn) {
+		String current = matrixResultSet[row1][numberColumn];
+		String parallel = matrixResultSet[row2][numberColumn];
+		boolean sort = false;
+		if (numeric) {
+			Long numberCurrent = Long.valueOf(current);
+			Long numberParallel = Long.valueOf(parallel);
+			if (numberCurrent < numberParallel) {
+				sort = true;
+			}
+		} else {
+			if (current.compareTo(parallel) < 0) {
+				sort = true;
+			}
+		}
+		if (sort) { //Executes the change of values
+			String[] temporal = matrixResultSet[row1];
+			matrixResultSet[row1] = matrixResultSet[row2];
+			matrixResultSet[row2] = temporal;
+			String rowTemporal = rowsCassandra.get(row2);
+			rowsCassandra.set(row2, rowsCassandra.get(row1));
+			rowsCassandra.set(row1, rowTemporal);
+		}
+	}
+
+	/**
+	 * Returns the ranges of values which are the same
+	 */
+	public List<Integer> rangesEqual(int beggining, int end, int columnNumber, String[][] matrixResultSet,
+			boolean numeric) {
+		List<Integer> rangesEqualValues = new ArrayList<>();
+		String numberRange = matrixResultSet[beggining][columnNumber];
+		rangesEqualValues.add(beggining);
+		if (beggining == end) {
+			rangesEqualValues.add(end);
+			return rangesEqualValues;
+		}
+		for (int i = beggining + 1; i <= end; i++) {
+			if (numeric) {
+				Long currentRange = Long.parseLong(numberRange);
+				Long currentValue = Long.parseLong(matrixResultSet[i][columnNumber]);
+				if (currentRange.longValue() != currentValue.longValue()) {
+					numberRange = matrixResultSet[i][columnNumber];
+					rangesEqualValues.add(i - 1);
+					rangesEqualValues.add(i);
+				}
+			} else {
+				String currentValue = matrixResultSet[i][columnNumber];
+				if (numberRange.compareTo(currentValue) != 0) {
+					numberRange = matrixResultSet[i][columnNumber];
+					rangesEqualValues.add(i - 1);
+					rangesEqualValues.add(i);
+				}
+			}
+
+		}
+		rangesEqualValues.add(end);
+		return rangesEqualValues;
+	}
+
+	/**
+	 * Compares two CSV files and returns true if they store the same data, false
+	 * otherwise
+	 */
+	public boolean compareCSV(String sqlPath, String cassandraPath, String nameTable) {
+		String introduction = "Table " + nameTable + ": ";
+		try (BufferedReader sql = new BufferedReader(new FileReader(sqlPath));
+			BufferedReader cassandra = new BufferedReader(new FileReader(cassandraPath));) {
+			String lineSQL;
+			String lineCassandra;
+			while ((lineSQL = sql.readLine()) != null) {
+				lineCassandra = cassandra.readLine();
+				VisualAssert va = new VisualAssert();
+				va.assertEquals(lineCassandra, lineSQL);
```

Comment:
This is not the intended use of Visual Assert. The goal is to simplify FILE comparisons, not line comparisons.
Most (or all) this code is not necessary, simply compare the string contents of actual and expected files.
I would recommend set the platform to JUNIT4 to view results from eclipse too.